__tests__/schema/thread/thread.ts

```diff
@@ -73,9 +74,14 @@ describe('uuid["threads"]', () => {
           threads: {
             nodes: [
               {
-                comments: { nodes: [{ id: comment1.id }, { id: comment2.id }] },
+                comments: {
+                  nodes: [
+                    { id: encodeThreadId(comment1.id) },
```

Comment:
All of them in this file should be reverted to what it was before.
```suggestion
                    { id: comment1.id },
```